client/components/pages/EditDocumentPage.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Link } from 'react-router';
+import toastr from 'toastr';
+import TinyMCE from 'react-tinymce';
+import FlatButton from 'material-ui/FlatButton';
+import SelectInput from '../forms/SelectInput.jsx';
+import TextInput from '../forms/TextInput.jsx';
+import Sidebar from '../layouts/Sidebar.jsx';
+import * as documentActions from '../../actions/documentActions';
+import handleError from '../../utils/errorHandler';
+
+class EditDocumentPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      document: Object.assign({}, props.document),
+      error: {},
+      saving: false
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEditorChange = this.handleEditorChange.bind(this);
+  }
+
+  componentDidMount() {
+    $('select').material_select();
+    $('#select-box').on('change', this.handleChange);
```

Comment:
Yea..initializing a select element in that manner is important if you are using materialize css